test/extended/util/annotate/generated/zz_generated.annotations.go

```diff
@@ -671,12 +671,6 @@ var Annotations = map[string]string{
 
 	"[sig-arch][Early] Managed cluster should [apigroup:config.openshift.io] start all core operators": " [Skipped:Disconnected] [Suite:openshift/conformance/parallel]",
 
-	"[sig-arch][Feature:ClusterUpgrade] Cluster should be upgradeable after finishing upgrade [Late][Suite:upgrade]": "",
-
-	"[sig-arch][Feature:ClusterUpgrade] Cluster should be upgradeable before beginning upgrade [Early][Suite:upgrade]": "",
```

Comment:
This is not related to our cleanup right? And the test still seems to be there:
https://github.com/openshift/origin/blob/2d41e5256e5642b858a2defa3894a9922f839e87/test/e2e/upgrade/upgrade.go#L142-L146

We might want to double check if we aren't disabling this unintentionally.

Reply:
good point, no idea how that got into it...

Reply:
okay that's a really weird issue, it's a completely different folder. On the master branch I don't have that change generated.

Reply:
okay that's a really weird issue, it's a completely different folder. On the master branch I don't have that change generated.